dev-docs/modules/chromeAiRtdProvider.md

```diff
@@ -0,0 +1,244 @@
+module_code : chromeAiRtdProvider
+enable_download : true
+sidebarType : 1
+---
+
+# Chrome AI RTD Provider
+
+## Overview
+
+The Chrome AI RTD Provider is a Prebid.js Real-Time Data (RTD) module that enhances bidding by leveraging Chrome's built-in AI capabilities. It can automatically detect page language using the [Chrome AI Language Detection API](https://developer.chrome.com/docs/ai/language-detection) and generate page summaries or keywords using the [Chrome AI Summarizer API](https://developer.chrome.com/docs/ai/summarizer-api). This information is added to the OpenRTB bid request objects, allowing bid adapters to optimize bids based on content language and context.
+
+## Features
+
+- Automatic language detection using the Chrome AI Language Detection API.
+- Automatic page summarization or keyword generation using the Chrome AI Summarizer API.
+- Caching of detected language and summaries/keywords in localStorage to reduce redundant API calls (configurable for summarizer).
+- Configurable options for both language detection (e.g., confidence threshold) and summarization (e.g., type, format, length).
+- Flexible ORTB2 path configuration for placing detected data.
+- Ability to enable/disable each feature independently.
+- Compatible with the Prebid.js RTD framework.
+
+## Integration
+
+### Build Setup
+
+To include the Chrome AI RTD Provider in your Prebid.js build, use the following command:
+
+```bash
+gulp build --modules=rtdModule,chromeAiRtdProvider
+```
+
+### Basic Integration
+
+Add the Chrome AI RTD Provider to your Prebid.js configuration:
+
+```javascript
+pbjs.setConfig({
+  realTimeData: {
+    dataProviders: [{
+      name: 'chromeAi',
+      waitForIt: true  // Optional: delays the auction until language detection completes
+    }]
+  }
+});
+```
+
+### Advanced Configuration
+
+Configure language detection and summarization with additional options:
+
+```javascript
+pbjs.setConfig({
+  realTimeData: {
+    dataProviders: [{
+      name: 'chromeAi',
+      waitForIt: true, // Set to true if auction should wait for both enabled features
+      params: {
+        languageDetector: {
+          enabled: true,       // Set to false to disable language detection
+          confidence: 0.9,     // Set minimum confidence threshold (0.0 - 1.0)
+          ortb2Path: 'site.content.language' // Default path for language
+        },
+        summarizer: {
+          enabled: false,      // Set to true to enable summarization/keyword generation
+          type: 'headline',    // 'headline','key-points', 'tldr' or 'teaser'
+          format: 'markdown', // 'plain-text' or 'markdown'
+          length: 'short',     // 'short', 'medium', or 'long'
+          ortb2Path: 'site.content.keywords', // Path for summary/keywords
+          cacheInLocalStorage: true // Whether to cache generated summary/keywords
+        }
+      }
+    }]
+  }
+});
+```
+
+## Configuration Options
+
+| Parameter | Scope | Type | Description | Default |
```

Comment:
```suggestion

{: .table .table-bordered .table-striped }
| Parameter | Scope | Type | Description | Default |
```

Adding stripes to the table 😁 

 https://deploy-preview-6174--prebid-docs-preview.netlify.app/dev-docs/modules/chromeairtdprovider